core/src/main/java/org/kohsuke/stapler/export/ExportConfig.java

```diff
@@ -38,4 +47,40 @@ public ExportConfig withClassAttribute(ClassAttributeBehaviour cab) {
         this.classAttribute = cab;
         return this;
     }
+
+    /**
+     * Controls serialization of {@link @Exported} properties.
```

Comment:
Syntax error. The `Jenkinsfile` is supposed to be validating that. I am trying to figure out why the build did not fail.